docker-compose.yml

```diff
@@ -176,6 +176,8 @@ services:
       - _APP_GRAPHQL_MAX_BATCH_SIZE
       - _APP_GRAPHQL_MAX_COMPLEXITY
       - _APP_GRAPHQL_MAX_DEPTH
+      - _APP_GITHUB_PRIVATE_KEY
```

Comment:
Might be cool to add a check for this connection on the Appwrite doctor and in the health API.